tests/test_dem.py

```diff
@@ -78,3 +79,17 @@ def test_buffer_extent():
     assert dem.buffer_extent(extent2, 0.1) == [-170, 53, -167, 55]
     assert dem.buffer_extent(extent2, 0.3) == [-170, 53, -167, 55]
     assert dem.buffer_extent(extent2, 0.4) == [-171, 52, -166, 56]
+
+
+def test_distance_meters_to_degrees():
+    assert dem.distance_meters_to_degrees(distance_meters=20, latitude=0) == (0.000179864321184, 0.000179864321184)
+    assert dem.distance_meters_to_degrees(distance_meters=20, latitude=45) == (0.000254366562405, 0.000179864321184)
+    assert dem.distance_meters_to_degrees(distance_meters=20, latitude=89.9) == (0.103054717208573, 0.000179864321184)
+    assert dem.distance_meters_to_degrees(distance_meters=20, latitude=-45) == (0.000254366562405, 0.000179864321184)
+    assert dem.distance_meters_to_degrees(distance_meters=20, latitude=-89.9) == (0.103054717208573, 0.000179864321184)
+    # The longitudenal distance in degrees will get larger and larger towards 90 degrees, approaching MAX_FLOAT.
+    # This is since cos(90) = 0, leading to a divide by zero issue.
+    with raises(ZeroDivisionError):
+        dem.distance_meters_to_degrees(20, 90)
+    with raises(ZeroDivisionError):
+        dem.distance_meters_to_degrees(20, -90)
```

Comment:
Add newline at end of file.
```suggestion
        dem.distance_meters_to_degrees(20, -90)

```